community.md

```diff
@@ -0,0 +1,143 @@
+---
+layout: default
+menu_id: community
+title: Trino Community
+---
+
+<div class="jumbotron card card-image" style="background-image: linear-gradient(to top, #001C93 0%, #000033 70%);background-size:cover;">
+  <div class="text-white text-center py-5 px-4">
+    <div>
+      <h1><a href="/community/">Community</a></h1>
+      <p class="lead">Join thousands of users and developers from all over the world. </p>
+    </div>
+  </div>
+</div>
+
+<div class="container">
+  <div class="col-md-12 spacer-60">
+    <div class="community-content-section">
+      <div class="community-content-container slack-content-container">
+        <div class="slack-text-container">
+          <h2>Join the discussion</h2>
+          <p>The Trino community is very active and helpful on Slack, with users and developers from all around the world. If you need help using or running Trino, this is the place to ask.</p>
+        </div>
+        <div class="slack-stats-container">
+          <div class="community-stats">
+            <span>3933</span>
+            Slack Members
+          </div>
+          <div class="community-stats">
+            <span>507</span>
+            Weekly Active
+          </div>
+          <div class="community-stats">
+            <span>454</span>
+            Contributors
+          </div>
+        </div>
+      </div>
+      <div class="slack-channels-top d-flex mb-3">
+        <a href="https://join.slack.com/t/trinodb/shared_invite/zt-h88upx5f-gVb8XUu16W_jo4I2Cw8ijA" class="slack-channel slack-blue-gradient">
+          <img src="../assets/images/graphics/community-general.svg">
+          Join Slack
+        </a>
+        <a href="https://trinodb.slack.com/archives/CFLB9AMBN" class="slack-channel slack-pink-gradient">
+          <i class="fa fa-home fa-lg"></i>
+          #general
+        </a>
+        <a href="https://trinodb.slack.com/archives/CGB0QHWSW" class="slack-channel slack-orange-gradient">
+          <i class="fa fa-tools fa-lg"></i>
+          #troubleshooting
+        </a>
+      </div>
+      <div class="slack-channels-bottom d-flex">
+        <a href="https://trinodb.slack.com/archives/CP1MUNEUX" class="slack-channel slack-orange-gradient">
+          <i class="fa fa-code fa-lg"></i>
+          #dev
+        </a>
+        <a href="https://trinodb.slack.com/archives/CFSDUNJ4W" class="slack-channel slack-pink-gradient">
+          <i class="fa fa-dharmachakra fa-lg"></i>
+          #kubernetes
```

Comment:
I would leave this channel out since we dont have any useful k8s offering from the project.